VL.Data/Services/SeedingService.cs

```diff
@@ -0,0 +1,122 @@
+﻿using Bogus;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using VL.Shared.Data;
+using VL.Shared.Interfaces;
+using VL.Shared.Models;
+
+namespace VL.Shared.Services
+{
+    public class SeedingService : ISeedingService
+    {
+        private readonly ApplicationDbContext _applicationDbContext;
+        private readonly Faker _faker;
+        private readonly UserManager<IdentityUser> _userManager;
+        private readonly IUserStore<IdentityUser> _userStore;
+        private readonly IUserEmailStore<IdentityUser> _emailStore;
```

Comment:
https://learn.microsoft.com/en-us/aspnet/core/security/authentication/identity-custom-storage-providers?view=aspnetcore-6.0
https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.identity.entityframeworkcore.userstore-1?view=aspnetcore-6.0
[3:52](https://troyweb.slack.com/archives/D03V5HYUQH5/p1664221959609429)
those are just stores for persisting data, not really things we typically interact with unless we have a custom thing going on